package-lock.json

```diff
@@ -1001,3599 +1001,233 @@
       "integrity": "sha512-inJlb0Idr6frKKVJNweG4t3BUbrvfSPhtwTuMzjLIUjSjEzu+/bNmSb4FrZcSfOGW9KCsvWY2pP4HMaoKzu86g=="
     },
     "@dcos/ui-kit": {
-      "version": "1.17.1",
-      "resolved": "https://registry.npmjs.org/@dcos/ui-kit/-/ui-kit-1.17.1.tgz",
-      "integrity": "sha512-0H1SDhPOcgxcfnM/N9E6n8JUnc8dtZ+ovg76h0Ndh9N+AKMmtOX3lDQKGLY77p5TnlvrFrHb0ovk2KXW1ZBuzQ==",
+      "version": "1.26.0",
+      "resolved": "https://registry.npmjs.org/@dcos/ui-kit/-/ui-kit-1.26.0.tgz",
+      "integrity": "sha512-xQqfv2mDzZjIVPwkaX2Hd/IeyCSCXma+R3xVLcTvWn4OqyX0Tx9UgoEVOfdazLtG6oUNGUH6ho3khv8n7KWyHg==",
       "requires": {
         "emotion": "9.2.12",
+        "emotion-theming": "9.2.9",
+        "focus-trap-react": "4.0.1",
         "memoize-one": "4.0.2",
-        "react": "16.5.2",
-        "react-dom": "16.5.2",
+        "react-click-outside": "3.0.1",
+        "react-delegate-component": "1.0.0",
         "react-emotion": "9.2.12",
+        "react-toggled": "1.2.7",
         "react-transition-group": "2.5.0",
         "react-virtualized": "9.20.1",
-        "semantic-release": "^15.9.6"
+        "semantic-release": "15.9.6"
       },
       "dependencies": {
-        "@octokit/rest": {
```

Comment:
Could be because of https://github.com/dcos-labs/ui-kit/commit/b687444e4a5230d829e9e9c51bcfb49d1e627b62#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

Reply:
Could be. I tested on master and the same thing happened - a lot of lines in package-lock were deleted.